splunk/tests/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM docker.io/splunk/splunk:latest
+
+ENV SPLUNK_START_ARGS=--accept-license
+ENV SPLUNK_PASSWORD=$SPLUNK_PASSWORD
+
+EXPOSE 8000
+EXPOSE 8089
+
+USER root
+
+COPY test_logs /opt/splunk/test_logs
+COPY indexes.conf /opt/splunk/etc/apps/search/local/indexes.conf
+COPY ../scripts/index_verification.sh /opt/splunk/index_verification.sh
+
+RUN microdnf install -y jq && \
+    microdnf clean all
+
+RUN /sbin/entrypoint.sh start-service &\
+    sleep 60 &&\
```

Comment:
Maybe loop and curl the service instead of blindly waiting a minute.

Reply:
Will do

Reply:
Done